modules/json/src/smithy4s/http/json/Cursor.scala

```diff
@@ -63,7 +63,12 @@ class Cursor private () {
       top -= 1
       list = stack(top) :: list
     }
-    throw PayloadError(PayloadPath(list), expecting, "Missing required field")
+    val path = PayloadPath(list)
+    throw PayloadError(
```

Comment:
let's change the `getMessage` implementation of the `PayloadError` class instead.

Reply:
Concatenating the path to the message field of the case class means that the information gets duplicated in the json output when the PayloadError is serialised. 

Showing the path makes sense in all scenarios.